src/apdu_sign.c

```diff
@@ -359,6 +359,7 @@ size_t handle_apdu_sign_with_hash(uint8_t instruction, volatile uint32_t *flags)
 int perform_signature(bool const on_hash, bool const send_hash) {
 #ifdef BAKING_APP
     write_high_water_mark(&G.parsed_baking_data);
+    ux_empty_screen();
```

Comment:
Each time an operation is being signed the `saverscreen` is redisplayed.
But in the specification (https://github.com/trilitech/ledger-app-tezos-baking/issues/21#issuecomment-1956295940), only baking operation should trigger the `saverscreen` and only if it is not activated.

Reply:
Added a global bool to make sure the screensaver is triggered only when it is not active.